tests/test_query.py

```diff
@@ -173,3 +173,20 @@ def test_query_search_with_quotes():
     sample_text = "I want to go ahead and cancel the booking"
     result = query(sample_text)
     assert result is False
+
+def test_emoji_matching():
+    """
+    Tests the query with emojis and matches it
+    """
+
+    # this should match
+    query = Query("😊")
+    sample_text = "Hello, how are you?😊"
+    result = query(sample_text)
+    assert result is True
```

Comment:
So this test looks to have failed again. 

When I debugged there I noticed it was because of the following piece of code in `eldar/entry.py`: `res = True if re.search(fr"\b{self.query}\b", doc) else False`.

Basically what is happening here is that since the regex pattern (`r"\b:smiling_face_with_smiling_eyes:\b"`) contains a word boundary at the end in the ":". The `\b` bit is causing it not to match. To fix this you're gonna have to add a check to see if the regex pattern (`self.query`) starts or ends with a word boundary/punctuation and handle is accordingly. probably an if\else